felix/fv/infrastructure/infra_etcd.go

```diff
@@ -56,6 +56,12 @@ func GetEtcdDatastoreInfra() (*EtcdDatastoreInfra, error) {
 		return nil, errors.New("failed to create etcd container")
 	}
 
+	arch := os.Getenv("ARCH")
```

Comment:
We use this block of code a few times - would be good to throw it in `felix/fv/utils/utils.go` so we don't need to duplicate it so much.

Reply:
Updated with PR: https://github.com/projectcalico/calico/pull/6989
Should I just abandon this PR?

Reply:
> We use this block of code a few times - would be good to throw it in `felix/fv/utils/utils.go` so we don't need to duplicate it so much.

I added a new func named 'GetSysArch' in utils.go, for your advice.